torch/_inductor/select_algorithm.py

```diff
@@ -2130,9 +2131,20 @@ def wait_on_futures():
                 timeout=precompilation_timeout_seconds,
             ):
                 if e := future.exception():
-                    log.error(
-                        "Exception %s for benchmark choice %s", e, futures[future]
+                    from torch._inductor.codegen.cuda.cuda_kernel import (
+                        CUDATemplateCaller,
                     )
+
+                    if isinstance(e, CUDACompileError) and isinstance(
+                        futures[future], CUDATemplateCaller
+                    ):
+                        log.debug(
+                            "Exception %s for benchmark choice %s", e, futures[future]
```

Comment:
```suggestion
                            "Exception %s for benchmark choice %s", e, futures[future], exc_info=True
```
Will ensure the same level of detail is printed as log.error